addons/website/views/snippets/s_experience.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_experience" name="Experience">
+    <section class="s_experience pt104 pb104">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="11">
+                <div class="o_grid_item g-height-8 g-col-lg-8 col-lg-6 align-content-center pt96 pb96" data-name="Box" style="z-index: 2; grid-area: 1 / 3 / 10 / 11; text-align: center;">
+                    <h1 class="display-3">Experience the<br/>best quality services</h1>
+                    <p>
+                        <br/>
+                        <br/>
+                        <a t-att-href="cta_btn_href" class="btn btn-lg btn-primary">Discover our services</a>
+                    </p>
+                </div>
+                <div class="o_grid_item oe_img_bg o_grid_item_image g-col-lg-4 g-height-10 col-lg-3 order-first pt96 pb96" style="z-index: 1; grid-area: 1 / 1 / 10 / 6;">
+                    <img
+                    class="img-fluid mx-auto"
+                    data-shape="web_editor/geometric_round/geo_round_pill"
+                    data-file-name="s_experience_default_image.jpg"
+                    data-original-mimetype="image/jpeg/"
+                    src="/web_editor/image_shape/website.s_experience_default_image/web_editor/geometric_round/geo_round_pill.svg"
+                    data-shape-colors=";;;;"
+                    />
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-2 g-height-9 d-none d-lg-block col-lg-3 pt96 pb96" style="z-index: 1; grid-area: 2 / 8 / 12 / 13;">
+                    <img
+                    class="img-fluid mx-auto"
+                    data-shape="web_editor/geometric_round/geo_round_pill"
+                    data-file-name="s_experience_default_image_2.jpg"
+                    data-original-mimetype="image/jpeg/"
+                    src="/web_editor/image_shape/website.s_experience_default_image_2/web_editor/geometric_round/geo_round_pill.svg"
+                    data-shape-colors=";;;;"/>
```

Comment:
Defining `data-shape-colors` seems not necessary, do you confirm? 
(same for the other image)